draft-ietf-tls-ctls.md

```diff
@@ -237,36 +237,53 @@ defined in {{RFC8446, Section 8.4}}.
 
 #### `dh_group`
 
-Value: a single `NamedGroup` ({{!RFC8446, Section 4.2.7}}) to use for key establishment.
+Value: a single `CTLSKeyShareGroup` to use for key establishment.
 
-This is equivalent to a literal "supported_groups" extension
-consisting solely of this group.
+~~~~
+struct {
+    NamedGroup group_name;
+    uint16 key_share_length;
+} CTLSKeyShareGroup;
+~~~~
+
+This is equivalent to a literal "supported_groups" extension in ClientHello
```

Comment:
Is it true that it is only present in ClientHello?

Reply:
[Apparently not](https://datatracker.ietf.org/doc/html/draft-rescorla-tls-rfc8446-bis-00#section-4.2)

```suggestion
This is equivalent to adding a "supported_groups" extension to every message
where that is allowed (i.e. ClientHello and EncryptedExtensions, in TLS 1.3) 
```

Reply:
Ah, I didn't know the existence of the table 1, thanks!